### ISSUE TITLE
Unidentifiable places

### ISSUE THREAD

**Issue by ChiaraPalladino** [ChiaraPalladino/furesearch#22]
![Image](https://github.com/ChiaraPalladino/furesearch/assets/168691206/889379ae-693c-44d0-bc19-014f5946e85c)

@ChiaraPalladino I cannot find good locations to tag "Πλινθινήτεω κόλπου", "Σερβωνίδος λίμνης", and "Κάσιον ὄρος". I'm pretty sure it is just describing the coast along the Red sea but I do not know how to tag the coast :'). What should I do?

_Originally posted by @Savannah-Bailey in https://github.com/ChiaraPalladino/furesearch/issues/21#issuecomment-2153279905_

**Comment by ChiaraPalladino** [comment]
I have opened a new issue because these are separate questions/discussions. 

I suspect Πλινθινήτεω κόλπου is Plinthinos Kolpos: https://pleiades.stoa.org/places/730147
Σερβωνίδος λίμνης is the lake serbonitis: https://pleiades.stoa.org/places/688025 
And here is Mount Casios: https://pleiades.stoa.org/places/687872

Were these not in ToposText?

**Comment by Savannah-Bailey** [comment]
Thank you, I found the latter two but Plinthinos Kolpos is labeled as an unlocated place so there is just a big question mark where the map should be.

**Comment by ChiaraPalladino** [comment]
Yeah, well if it's unlocated there's nothing we can do :) You should still link it to the pleiades entry, even though it is not going to appear on the map.

**Comment by Savannah-Bailey** [comment]
Ok! Will do 🫡. I have another question about tagging but it in kinda unrelated to this so I am going to open up another issue if thats ok.

**Comment by Savannah-Bailey** [comment]
In Chapter 17 Herodotus is naming all of the  different stoma of the Nile and I could find a location for all of them except for "Βουκολικὸν" in section 6.

**Comment by Savannah-Bailey** [comment]
In Chapter 50, section 2, Herodotus talks about "Πελασγῶν" and I was not sure what to assign as the location in addition to the collective and ethnic tags.

**Comment by Savannah-Bailey** [comment]
@ChiaraPalladino In chapter 91, Section 1 I did not know where to place the location pin for Νέης πόλιος. Should I leave it flagged or is there a location I can pin it as?

**Comment by ChiaraPalladino** [comment]
> @ChiaraPalladino In chapter 91, Section 1 I did not know where to place the location pin for Νέης πόλιος. Should I leave it flagged or is there a location I can pin it as?

Looked into it. There is no suitable match for this place. Keep it flagged.
